<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoCenterWidget</class>
 <widget class="QWidget" name="InfoCenterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1002</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1002</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EntCenterWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#EntCenterWidget{background-color: rgb(255, 255, 255);};</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTop" native="true">
     <property name="maximumSize">
      <size>
       <width>1002</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 252, 252);</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>2</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgba(222,222,222,1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBack">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>15</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="EntCenterWidget.qrc">
        <normaloff>:/EntCenterWidget/image/min.png</normaloff>:/EntCenterWidget/image/min.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelSubTitle">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>15</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="text">
       <string>标题</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>15</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEntCenter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#1E8BED;</string>
      </property>
      <property name="text">
       <string>企业中心</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUserCenter">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>15</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#000000;</string>
      </property>
      <property name="text">
       <string>个人中心</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelEntCenter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>120</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid #1E8BED;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelUserCenter">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>55</y>
        <width>120</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid #1E8BED;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetCenter" native="true">
     <property name="maximumSize">
      <size>
       <width>1002</width>
       <height>620</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px lineed rgba(222,222,222,1);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
